components/Splash/SplashScreen.tsx

```diff
@@ -0,0 +1,21 @@
+import React, { useEffect, useState } from "react";
+import { ActivityIndicator, View } from "react-native";
+import { styles } from "./style";
+
+export const SplashScreen = ({ navigation}: {navigation: any}) => {
```

Comment:
Cool fake Splash Screen. This seems to work.

Reply:
Thought a little spin would give a feel of authentication.